Dockerfile

```diff
@@ -10,9 +11,8 @@ RUN go mod download
 
 COPY cmd cmd
 COPY internal internal
-COPY pkg pkg
 
-RUN go build -o /server ./cmd/main.go
+RUN go build -o /server -tags=nomsgpac ./cmd/main.go
```

Comment:
Please add `k` to the end of the tag name. It seems not right, because the tag should be `nomsgpack`

Reply:
(: